tasking/pulp/tasking/celery_app.py

```diff
@@ -69,6 +69,8 @@ def initialize_worker(sender, instance, **kwargs):
     :param kwargs:   Other params (unused)
     :type  kwargs:   dict
     """
+    from pulp.tasking.services.worker_watcher import delete_worker
```

Comment:
It is so ^ because `delete_worker` imports models and also it imports `cancel` from `util` which also imports models.